src/actinia_core/processing/actinia_processing/ephemeral/download_cache_management.py

```diff
@@ -59,37 +58,30 @@ def __init__(self, *args):
     def _execute(self):
         self._setup()
 
-        if os.path.exists(self.user_download_cache_path) and os.path.isdir(
-            self.user_download_cache_path
-        ):
-            executable = "/usr/bin/du"
-            args = ["-sb", self.user_download_cache_path]
+        if not os.path.exists(self.user_download_cache_path):
```

Comment:
`os.path.exists` checks only if the path exists, but not if it is a directory or not.

Reply:
True. If there is no directory and no file it will create the folder. Do you think I should check if it exists if it is a file and then raise an error?

Reply:
yes. Otherwise the next step would fail, or not?

Reply:
No it would be fine:
```
/usr/bin/du -sb download_cache_management.py 
2838	download_cache_management.py
```
Although it would indicate a config error. But an error that the download cache is not configured correctly wouldn't help a user much at this point. If it is important to check it would make more sense somewhere else, e.g. on actinia startup where the person installing actinia would be made aware of it. Do you think that should be added here in this PR? Or new feature request?